lib/option.rb

```diff
@@ -85,7 +85,7 @@ def display_name
     VmSize.new("burstable-#{it}", "burstable", it, it * 50, it * 50, (it * 1.6).to_i, storage_size_options, io_limits, false, "arm64")
   }).concat(AWS_FAMILY_OPTIONS.product([2, 4, 8, 16, 32, 64]).map { |family, vcpu|
     memory_coefficient = (family == "c6gd") ? 2 : 4
-    arch = (family == "m6id") ? "x64" : "arm64"
+    arch = family.include?("g") ? "arm64" : "x64"
```

Comment:
I think it would be better to explicitly list families here. "g" might be in a different place of the family name for a different reason.